shared-module/src/components/Forum/Thread.tsx

```diff
@@ -0,0 +1,198 @@
+/* eslint-disable i18next/no-literal-string */
```

Comment:
Don't disable for the whole file and use the translate function for the strings.